app.yaml

```diff
@@ -59,6 +59,13 @@ handlers:
   static_files: core/templates/dev/head/\1
   upload: core/templates/dev/head/(.*\.(js))$
   secure: always
+- url: /templates/dev/head/components/(.*\.(html))$
```

Comment:
(From Ben) Should we set cache control settings on these HTML files now that they're being cached?